soyuz_app/library/slack.py

```diff
@@ -26,33 +27,35 @@ def create_channel(self, section, channel_name):
         except SlackApiError as e:
             logger.error("Error creating conversation: {}".format(e))
 
-        # set and update slack_channel_id
-        section.slack_channel_id = result["channel"]["id"]
-        section.save()
+        else:
+            # set and update slack_channel_id
+            section.slack_channel_id = result["channel"]["id"]
+            section.save()
 
-    # no longer needed
+    # no longer needed after b13, slack registration is automated using webhook
 
     def lookup_by_email(self, user, user_list):
         try:
             email_lookup_result = self.client.users_lookupByEmail(email=user.email)
 
         except SlackApiError as e:
             logger.error("Error looking up email: {}".format(e))
```

Comment:
you probably don't want to log an error, unless the error is something besides user not found, right?